conda/recipes/libkvikio/meta.yaml

```diff
@@ -28,33 +24,61 @@ build:
     - SCCACHE_BUCKET=rapids-sccache
     - SCCACHE_REGION=us-west-2
     - SCCACHE_IDLE_TIMEOUT=32768
```

Comment:
iirc we aren't supposed to specify values for these keys. Those come from the parent environment.
```suggestion
    - SCCACHE_BUCKET
    - SCCACHE_REGION
    - SCCACHE_IDLE_TIMEOUT
```